### DISCUSSION TITLE
Shrink

### DISCUSSION THREAD

**Discussion by AhmadrezaHamidi** [community/community#50053] [Pull Requests] [answered]
### Select Topic Area

Question

### Body

I got the sub on github acivment pull shrink, but after a few hours GitHub took it from me, why?

**Comment by Anmol-Baranwal** [comment, marked as answer]
There are a few reasons why this might happen:

1. Achievement was earned by mistake: It is possible that you earned the achievement by mistake or through an unintended action. GitHub may have detected this and removed the achievement.

2. Glitch or bug: There may have been a glitch or bug in the GitHub system that caused the achievement to be awarded incorrectly. In this case, GitHub may have later corrected the mistake by removing the achievement.

3. Violation of GitHub policies: The achievement may have been removed because it was found that it violated GitHub's policies. For example, if the achievement was earned through the use of a third-party tool or service that violates GitHub's terms of service, then it may have been removed.

If you're still unsure why your achievement was taken away, I recommend contacting [GitHub support](https://support.github.com/contact?tags=docs-generic) directly. They will be able to provide you with more information and help you resolve the issue.

**Reply by AhmadrezaHamidi** [reply]
🥲🥹

**Reply by Anmol-Baranwal** [reply]
@vestaabner 
I just wanted to check in and see if you had any further questions or concerns about my answer on Github. If you have no further doubts or if your doubts have been resolved, would you mind marking my answer as accepted?

It would be greatly appreciated and it will also help other users who might be looking for a solution to a similar issue.

**Reply by AhmadrezaHamidi** [reply]
thnks
